Lesson32/selcan/my-app/src/shared-components/typography/HeadingS/index.jsx

```diff
@@ -0,0 +1,8 @@
+import './styles.css';
+
+export const HeadingS = ({title, bold}) => {
+
+  return (
+    <h2 className={bold ? 'bold' : ''}>{title}</h2>
```

Comment:
HeadingM and Heading S would actually have different importance. So you can assume that the largest heading (Display) is H1, HeadingL is h2, HeadingM is h3, HeadingS is h4. However, I recommend you to add class to every element, not just use element selector as in React it may be hard to predict which CSS selector would actually be used (code from styles for HeadingM and for HeadingS would be merge into one "file").